tests/test_elastic.py

```diff
@@ -49,6 +50,23 @@ def query_user(name):
     return query(term)
 
 
+def retry_assertion(interval=0.3
```

Comment:
Probably so. Likely we should to a full audit of all tests that use elasticsearch, as they are all subject to this issue.